agent/engine/engine_integ_test.go

```diff
@@ -507,9 +507,11 @@ func TestLabels(t *testing.T) {
 	assert.EqualValues(t, "", state.Config.Labels["label1"])
 
 	// Kill the existing container now
-	taskUpdate := *testTask
```

Comment:
https://github.com/aws/amazon-ecs-agent/blob/master/agent/engine/engine_unix_integ_test.go noticed there are other tests that do this too here. any reason why they aren't flakey? should we change those too?

Reply:
I want to monitor for that. I've changed all occurrences in this file since we have 2 of them being flakey, and see if it should be applied generally throughout our integ tests.